apps/prairielearn/src/migrations/20240126125616_group_roles__can_assign_roles__add.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE group_roles
+ADD COLUMN IF NOT EXISTS can_assign_roles BOOLEAN NOT NULL DEFAULT FALSE;
+
+ALTER TABLE group_roles
+DROP COLUMN IF EXISTS can_submit_assessment;
```

Comment:
I don't think this is safe to drop until we've deployed the change to `GroupRoleSchema` everywhere. Otherwise, if we try to validate `SELECT group_roles.*`, I think we'll get Zod complaining about "expected number or null, got undefined".

Reply:
Ah, good point. I'll restore it for now.